2020-10-25T10:22:03.2152100+01:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-10-25T10:22:03.2539440+01:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-25T10:22:03.2593440+01:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-25T10:22:03.2730560+01:00  [INF] Content root path: "/Users/nikolasekiw/Prog-prosjekter/ITPE3200 21. Kunde-Rest/Kunde-SPA" (b5d60022)
2020-10-25T13:43:40.1641050+01:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-10-25T13:43:40.1697890+01:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2020-10-25T13:43:40.1701030+01:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-25T13:43:40.1701370+01:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-25T13:43:40.1701700+01:00  [INF] Content root path: "/Users/nikolasekiw/Prog-prosjekter/ITPE3200 21. Kunde-Rest/Kunde-SPA" (b5d60022)
2020-10-25T13:46:35.5241610+01:00 0HM3OSECQVOA3:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.Net.WebSockets.WebSocketException (0x80004005): The WebSocket is in an invalid state ('Aborted') for this operation. Valid states are: 'Open, CloseReceived'
   at System.Net.WebSockets.WebSocketValidate.ThrowIfInvalidState(WebSocketState currentState, Boolean isDisposed, WebSocketState[] validStates)
   at System.Net.WebSockets.ManagedWebSocket.CloseOutputAsyncCore(WebSocketCloseStatus closeStatus, String statusDescription, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PumpWebSocket(WebSocket source, WebSocket destination, Int32 bufferSize, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.AcceptProxyWebSocketRequest(HttpContext context, Uri destinationUri, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
2020-10-25T13:46:35.5269330+01:00 0HM3OSECQVOA3:00000001 [WRN] The response has already started, the error page middleware will not be executed. (e1dd4c0d)
2020-10-25T13:46:35.5359070+01:00 0HM3OSECQVOA3:00000001 [ERR] Connection id ""0HM3OSECQVOA3"", Request id ""0HM3OSECQVOA3:00000001"": An unhandled exception was thrown by the application. (560e7d32)
System.Net.WebSockets.WebSocketException (0x80004005): The WebSocket is in an invalid state ('Aborted') for this operation. Valid states are: 'Open, CloseReceived'
   at System.Net.WebSockets.WebSocketValidate.ThrowIfInvalidState(WebSocketState currentState, Boolean isDisposed, WebSocketState[] validStates)
   at System.Net.WebSockets.ManagedWebSocket.CloseOutputAsyncCore(WebSocketCloseStatus closeStatus, String statusDescription, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PumpWebSocket(WebSocket source, WebSocket destination, Int32 bufferSize, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.AcceptProxyWebSocketRequest(HttpContext context, Uri destinationUri, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.SpaServices.Extensions.Proxy.SpaProxy.PerformProxyRequest(HttpContext context, HttpClient httpClient, Task`1 baseUriTask, CancellationToken applicationStoppingToken, Boolean proxy404s)
   at Microsoft.AspNetCore.Builder.SpaProxyingExtensions.<>c__DisplayClass2_0.<<UseProxyToSpaDevelopmentServer>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
